libsignal-service-hyper/src/push_service.rs

```diff
@@ -413,6 +442,34 @@ impl PushService for HyperPushService {
         Self::json(&mut response).await
     }
 
+    #[tracing::instrument(skip(self))]
+    async fn post(
+        &mut self,
+        service: Endpoint,
+        path: &str,
+        additional_headers: &[(&str, &str)],
+        credentials_override: HttpAuthOverride,
+    ) -> Result<HashMap<String, String>, ServiceError> {
```

Comment:
Post without any data?

Reply:
yes : 😁

Reply:
In that case maybe
```suggestion
    #[tracing::instrument(skip(self))]
    async fn post_empty_body(
        &mut self,
        service: Endpoint,
        path: &str,
        additional_headers: &[(&str, &str)],
        credentials_override: HttpAuthOverride,
    ) -> Result<HashMap<String, String>, ServiceError> {
```
just for clarity?

Reply:
In that case maybe
```suggestion
    #[tracing::instrument(skip(self))]
    async fn post_empty_body(
        &mut self,
        service: Endpoint,
        path: &str,
        additional_headers: &[(&str, &str)],
        credentials_override: HttpAuthOverride,
    ) -> Result<HashMap<String, String>, ServiceError> {
```
just for clarity?